packages/api/src/clickhouse/index.ts

```diff
@@ -1096,14 +1097,36 @@ const buildEventSeriesQuery = async ({
       ? buildSearchColumnName(propertyTypeMappingsModel.get(field), field)
       : '';
 
-  const hasGroupBy = groupBy != '' && groupBy != null;
   const isCountFn = aggFn === AggFn.Count;
-  const groupByField =
-    hasGroupBy &&
-    buildSearchColumnName(propertyTypeMappingsModel.get(groupBy), groupBy);
+  const groupByFields: string[] = [];
+  for (const groupBy of groupBys) {
```

Comment:
this feels like we should just do a `.map` and throw if the column name can't be generated (otherwise we'd get a non-sensical group value right?)

Reply:
currently those invalid groups are skipped. I guess we can throw

Reply:
I think it'd be problematic if it's a silent failure, because the only way we can map back the group values with the fields is by array index, and if the array index is off it can get weird